client/src/main/java/com/linecorp/decaton/client/DecatonClient.java

```diff
@@ -46,6 +49,15 @@
      */
     CompletableFuture<PutTaskResult> put(String key, T task, long timestamp);
 
+    /**
+     * Put a task onto associated decaton queue with specifying arbitrary delay duration.`
```

Comment:
"specifying arbitrary delay duration." => "specifying some fields of task metadata."